test/dpul_collections_web/live/home_live_test.exs

```diff
@@ -22,19 +23,31 @@ defmodule DpulCollectionsWeb.HomeLiveTest do
              {:error, {:live_redirect, %{kind: :push, to: "/search?q=cats"}}}
   end
 
-  test "recent item blocks", %{conn: conn} do
-    Solr.add(SolrTestSupport.mock_solr_documents(10), active_collection())
-    Solr.commit(active_collection())
+  describe "recent item blocks" do
+    test "renders 3 cards", %{conn: conn} do
+      Solr.add(SolrTestSupport.mock_solr_documents(10), active_collection())
+      Solr.commit(active_collection())
+
+      {:ok, _view, html} = live(conn, "/")
+
+      links =
+        html
+        |> Floki.parse_document!()
+        |> Floki.find(".item-link")
+        |> Enum.flat_map(fn a -> Floki.attribute(a, "href") end)
+
+      assert Enum.count(links) == 3
+    end
 
-    {:ok, _view, html} = live(conn, "/")
+    test "link to recently added", %{conn: conn} do
+      {:ok, view, _} = live(conn, "/")
 
-    links =
-      html
-      |> Floki.parse_document!()
-      |> Floki.find(".item-link")
-      |> Enum.flat_map(fn a -> Floki.attribute(a, "href") end)
+      assert view
+             |> element("#recently-added-link")
+             |> render_click()
 
-    assert Enum.count(links) == 3
+      assert_redirected(view, "/search?sort=recently_added")
```

Comment:
The argument's `sort_by` I wonder if there was a way to validate that.

Reply:
good catch.